public/index.html

```diff
@@ -372,7 +372,7 @@ <h3 class="pl-1 fontHeading">Create New Task</h3>
                 </div>
                 <div class="form-group col-md-4">
                     <label for="inputState">Sprint Number</label>
-                    <input type="number" class="form-control" id="createNewTaskSprintNumber">
+                    <input type="number" class="form-control" id="sprintNumber">
```

Comment:
Here you need not to change the id because this belongs to create new task

Reply:
updated